lib/utils.nix

```diff
@@ -151,16 +151,18 @@ rec {
   toLuaLongLiteral =
     string:
     let
-      buildInfix =
+      buildTokens =
```

Comment:
We could debate the naming here, I wonder if `findTokens` and `findInfix` make more sense, since we are essentially _searching for_ a closing token that doesn't conflict with the string?

Not a big deal and doesn't blosk merging.

Reply:
I agree, I prefer "find" over "build".

I don't think we have a function that should be called `findInfix`, though. We now have a `findToken` (for vimscript) and `findTokens` (for lua).